app/models/reports/repository.rb

```diff
@@ -155,17 +163,46 @@ def initialize(regions, periods:)
     end
 
     memoize def controlled
+      return controlled_v2 if reporting_schema_v2?
+      science("controlled") do |e|
+        e.use { controlled_v1 }
+        e.try { controlled_v2 }
+        e.raise_on_mismatches = true
+      end
+    end
+
+    def controlled_v1
       region_period_cached_query(__method__) do |entry|
         control_rate_query.controlled(entry.region, entry.period).count
       end
     end
 
+    def controlled_v2
+      regions.each_with_object({}).each do |region, hsh|
+        hsh[region.slug] = control_rate_query_v2.controlled_counts(region, range: active_range(region)).tap { |hsh| hsh.default = 0 }
+      end
+    end
+
     memoize def uncontrolled
+      return uncontrolled_v2 if reporting_schema_v2?
+      science("uncontrolled") do |e|
+        e.use { uncontrolled_v1 }
+        e.try { uncontrolled_v2 }
```

Comment:
The fact that you're not raising here makes me think the other one was left by oversight.